src/uct/ib/mlx5/gga/gga_mlx5.c

```diff
@@ -625,6 +625,22 @@ uct_gga_mlx5_iface_is_reachable_v2(const uct_iface_h tl_iface,
            uct_ib_iface_is_reachable_v2(tl_iface, params);
 }
 
+static int
+uct_gga_mlx5_ep_is_connected(uct_ep_h tl_ep,
+                             const uct_ep_is_connected_params_t *params)
+{
+    uint32_t addr_qp = 0;
+    uct_gga_mlx5_ep_address_t *ep_addr;
+
+    if (params->field_mask & UCT_EP_IS_CONNECTED_FIELD_EP_ADDR) {
+        ep_addr = (uct_gga_mlx5_ep_address_t*)params->ep_addr;
+        addr_qp = uct_ib_unpack_uint24(ep_addr->qp_num);
+    }
+
+    return uct_rc_mlx5_ep_is_connected_to_addr(tl_ep, params, addr_qp) &&
+           uct_base_ep_is_connected(tl_ep, params);
```

Comment:
let's align with other transports which use `uct_base_ep_is_connected`:
```suggestion
    uct_gga_mlx5_ep_address_t *ep_addr;
    uint32_t addr_qp;

    if (!(params->field_mask & UCT_EP_IS_CONNECTED_FIELD_EP_ADDR)) {
        return 0;
    }

    if (!uct_base_ep_is_connected(tl_ep, params)) {
        return 0;
    }

    ep_addr = (uct_gga_mlx5_ep_address_t*)params->ep_addr;
    addr_qp = uct_ib_unpack_uint24(ep_addr->qp_num);
    return uct_rc_mlx5_ep_is_connected_to_addr(tl_ep, params, addr_qp);
```

Reply:
ep_addr is optional, so cannot use:
```
    if (!(params->field_mask & UCT_EP_IS_CONNECTED_FIELD_EP_ADDR)) {
        return 0;
    }
```

Reply:
is not `addr_qp == 0` means `false` for `is_connected` state?

Reply:
it means we skip the comparison between addr_qp and qp_num (and only compare device address)

Reply:
ok